lib/modules/manager/fleet/extract.spec.ts

```diff
@@ -61,6 +64,49 @@ kind: Fleet
         ]);
       });
 
+      it('should parse valid configuration with target customization', () => {
+        const result = extractPackageFile(
+          validFleetYamlWithCustom,
+          'fleet.yaml'
+        );
+
+        expect(result).not.toBeNull();
+        expect(result?.deps).toMatchObject([
+          {
+            currentValue: 'v1.8.0',
+            datasource: 'helm',
+            depName: 'cert-manager',
+            packageName: 'cert-manager',
+            registryUrls: ['https://charts.jetstack.io'],
+            depType: 'fleet',
+          },
+          {
+            currentValue: 'v1.9.2',
+            datasource: 'helm',
+            depName: 'rke2',
```

Comment:
i think this different depName will confuse users.

Reply:
🤔 My reasoning to implement it like this is, that users will for sure want separate PRs for each customization. At least by default, but maybe there is a better way to do this.

Reply:
I am ok with the proposed change